fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -200,7 +200,8 @@ def build_download_command(username, password, apple_id, destination = "/tmp")
     end
 
     def build_upload_command(username, password, source = "/tmp")
-      [
+      short_team_id = (ENV['FASTLANE_SHORT_TEAM_ID'] || '').strip
```

Comment:
Setting this value using environment variables is not the right way to solve this problem I think. Instead add a new optional parameter to this method to let the user specify the team. Also I guess you have the same parameter for the download command, right?